README_en-US.md

```diff
@@ -0,0 +1,60 @@
+# Rusbé
+
+![GitHub package.json version](https://img.shields.io/github/package-json/v/erick2280/rusbe-web-client)
+
+Rusbé is a follow up platform of the University Restaurant of UFPE.
+
+## Setup
+
+### Configurando
```

Comment:
I think this line "Configurando" should be removed.

Reply:
Yes, my mistake, it has already been fixed.